pkgs/development/python-modules/Theano/theano-with-cuda/default.nix

```diff
@@ -56,7 +56,7 @@ buildPythonPackage rec {
     pycuda
     cudatoolkit
     libgpuarray
-  ] ++ (stdenv.lib.optional (cudnn != null) [ cudnn ]);
+  ] ++ (stdenv.lib.optional (cudnn != null) cudnn);
```

Comment:
I don't know why it would have been optional.  From the blame history it seems that is just how @artuuge wrote it in the original commit.  I can certainly adjust the pull to just include it unconditionally.